src/shared/ui/section-label/section-label.tsx

```diff
@@ -1,9 +1,49 @@
-import './section-label.scss';
+import { HTMLAttributes } from 'react';
+import { type VariantProps, cva } from 'class-variance-authority';
+import classNames from 'classnames/bind';
 
-type SectionLabelProps = {
-  label: string;
-};
+import styles from './section-label.module.scss';
+
+type SectionLabelProps = HTMLAttributes<HTMLDivElement> &
+  VariantProps<typeof sectionLabelVariants> & { label: string };
+
+export const cx = classNames.bind(styles);
 
-export const SectionLabel = ({ label }: SectionLabelProps) => (
-  <div className="section-label">{label}</div>
-);
+const sectionLabelVariants = cva(cx('label'), {
+  variants: {
+    marginSize: {
+      small: cx('margin-small'),
+      medium: cx('margin-medium'),
+      large: cx('margin-large'),
+    },
+    fontSize: {
+      small: cx('font-small'),
+      large: cx('font-large'),
+    },
+  },
+  defaultVariants: {
+    marginSize: 'large',
+    fontSize: 'large',
+  },
+});
+
+export const SectionLabel = ({
+  label,
+  marginSize,
+  fontSize,
+  className,
+  ...props
+}: SectionLabelProps) => {
+  return (
+    <div
```

Comment:
We try to use semantic tags wherever possible.

Reply:
> We try to use semantic tags wherever possible.

I initially wanted to use a span, but since we have margin, I abandoned this idea. Would it be better to use a <label> tag?

Reply:
Let's use tag `<p><p />`.

Reply:
Maybe <h>?

Reply:
And what number of h? The font of 14px is in Body in Figma, not in headings

Reply:
I'd agree with @NikitaStarmoussov